AppCenter/AppCenter/Internals/Sender/MSHttpSender.m

```diff
@@ -24,6 +24,16 @@ - (id)initWithBaseUrl:(NSString *)baseUrl
          queryStrings:(NSDictionary *)queryStrings
          reachability:(MS_Reachability *)reachability
        retryIntervals:(NSArray *)retryIntervals {
+  return [self initWithBaseUrl:baseUrl apiPath:apiPath headers:headers queryStrings:queryStrings reachability:reachability retryIntervals:retryIntervals maxNumberOfConnections:4];
```

Comment:
It's actually only '1' max connections for OC if we refer to the doc but maybe I am not uptodate.

Reply:
It's two - @heya-msft gave feedback on this.